src/collective/cover/static/contentchooser.js

```diff
@@ -1,18 +1,40 @@
 (function ($) {
     var ajaxSearchRequest = [];
-    function contentSearchFilter(url) {
+    var timeoutIDs = [];
+    function contentSearchFilter(b_start) {
```

Comment:
sorry @hvelarde I missed your pré-review of this code, and don't remember what should I change here..